docs/current_docs/features/caching.mdx

```diff
@@ -20,3 +20,13 @@ Layer caching involves caching the layers of container images as they are built
 Volume caching involves caching specific parts of the filesystem and reusing them on subsequent function calls if they are unchanged. This is especially useful when dealing with package managers such as `npm`, `maven`, `pip` and similar. Since these dependencies are usually locked to specific versions in the application's manifest, re-downloading them on every session is inefficient and time-consuming.
 
 For these tools to cache properly, they need their own cache data (usually a directory) to be persisted between sessions. By using a cache volume for this data, Dagger can reuse the cached contents across pipeline runs and reduce execution time.
+
+## Cache size and pruning
+
+The layer cache size configuration is managed by [Buildkit's garbage collector](https://github.com/moby/buildkit/blob/master/docs/buildkitd.toml.md?plain=1#L83-L94).
+
+To free up disk space used by the cache, use the following command:
```

Comment:
Which cache does this impact?

Reply:
I would also love a small table mapping the `type` of the filters to dagger's functions/apis/etc.